apps/demo-react/components/WalletInfo.tsx

```diff
@@ -1,18 +1,28 @@
-import { useConnectorInfo } from 'reef-knot';
-import { useWeb3 } from '@reef-knot/web3-react';
+import { useWeb3, useConnectorInfo } from 'reef-knot/web3-react';
+import { useAccount } from 'wagmi';
 
 const WalletInfo = () => {
   const connectorInfo = useConnectorInfo();
-  const { account } = useWeb3();
+  const web3Info = useWeb3();
+  const { address: wagmiAddress, status: wagmiStatus } = useAccount();
```

Comment:
Yes, I wanted to explicitly show that they are coming from wagmi. Some wallets connections are detected by web3-react only, some are by wagmi only, and I want to know when it happens. Hopefully, web3-react will be fully removed soon and this will not be needed when it happens.